cdk_infra/lib/stacks/config-map-providers/config-map-providers-stack.ts

```diff
@@ -0,0 +1,47 @@
+import { Duration, Stack, StackProps } from 'aws-cdk-lib';
+import {
+  Effect,
+  PolicyStatement,
+  AccountRootPrincipal
+} from 'aws-cdk-lib/aws-iam';
+import { Bucket, BucketEncryption } from 'aws-cdk-lib/aws-s3';
+import { Construct } from 'constructs';
+
+export interface ConfigMapProvidersStackProps extends StackProps {
+  // Suffix used to test the stack and avoid collisions with s3 buckets names, that must be globally unique.
+  suffix?: string;
+}
+
+/**
+ * Stack containing resources used by config map providers in the integration
+ * tests.
+ */
+export class ConfigMapProvidersStack extends Stack {
+  constructor(
+    scope: Construct,
+    id: string,
+    props: ConfigMapProvidersStackProps
+  ) {
+    super(scope, id, props);
+
+    const suffix = props.suffix || '';
+    const bucketName = `aws-otel-collector-integ-test-configurations${suffix}`;
```

Comment:
```suggestion
    const bucketName = `adot-collector-integ-test-configurations${suffix}`;
```

Can we move away from the `aws-otel` nomenclature for all new infrastructure.